gcp-ts-gke-knative/README.md

```diff
@@ -0,0 +1,39 @@
+# gcp-ts-gke-knative
+All-in-one demo of Knative on GKE
+
+**Not intended for production use**
+
+This demo was adapted from the 
+[Knative installation guide](https://www.knative.dev/docs/install/knative-with-gke/#creating-a-kubernetes-cluster)
+for GKE and the related
+[example app](https://www.knative.dev/docs/install/getting-started-knative-app/)
+
+**Important Note**
+This demo creates a 3-node GKE cluster with `n1-standard-4` instances.
+Remember to tear down these resources after use to avoid unwanted charges.
+
+## Prerequisites
+
+* GCP account configured with the `Kubernetes Engine Cluster Admin` role
+
+## Known issues:
+
+1. Stack operations get very slow with so many resources in one stack.
```

Comment:
I think we probably want to fix this (or fix the product!) before publishing the example.  This doesn't feel like it's putting a good experience in front of users.

Reply:
+1 on fixing the product. I wasn't sure if it was worth the effort to split up in stacks for this demo.